components/suggestion/emoticon_provider.jsx

```diff
@@ -10,7 +10,7 @@ import {getEmojiMap, getRecentEmojis} from 'selectors/emojis';
 
 import store from 'stores/redux_store.jsx';
 
-import * as Emoticons from 'utils/emoticons.jsx';
+import * as Emoticons from 'utils/emoticons.tsx';
```

Comment:
same here

Reply:
Please remove the extension for consistency.

Reply:
@guigui64 If you don't mind, I'll push commit so we can merge.